docs/conf.py

```diff
@@ -46,11 +50,16 @@
 # -- General configuration ---------------------------------------------------
 
 # We want to make sure all the following warnings fail the build
-warnings.filterwarnings("error", category=SunpyDeprecationWarning)
+# warnings.filterwarnings("error", category=SunpyDeprecationWarning)#Feb272025 - Create new PR to fix
 warnings.filterwarnings("error", category=SunpyPendingDeprecationWarning)
 warnings.filterwarnings("error", category=MatplotlibDeprecationWarning)
 warnings.filterwarnings("error", category=AstropyDeprecationWarning)
 
+# TEMPORARY FIX: Suppress SunPy Deprecation Warnings for CTYPE1 ('solar-x')
+warnings.filterwarnings(
+    "ignore", category=SunpyDeprecationWarning
+)  # Feb272025 - Create new PR to fix
+
```

Comment:
Since #330 was merged, these warnings filters might no longer be necessary since we put a temporary upper limit on the version of SunPy for documentation builds.

Reply:
Rights, remove comments. Thanks Nic!